src/applications/check-in/day-of/pages/Landing.jsx

```diff
@@ -76,7 +78,8 @@ const Landing = props => {
         goToErrorPage();
       }
 
-      if (token) {
+      if (token && !sessionCallMade) {
```

Comment:
Do you need add this to the `useEffect`  dependency array?

Reply:
VSCode complained when I didn't.

Reply:
Oh whoops yes I added it to one but not the other, thanks for the catch